rest-api-spec/src/main/resources/rest-api-spec/api/nodes.reload_secure_settings.json

```
@@ -5,6 +5,9 @@
       "description":"Reloads secure settings."
     },
     "stability":"stable",
+    "headers":{
+      "accept": [ "application/json"]
```

```suggestion
      "accept": [ "application/json"],
      "content_type": [ "application/json" ]
```

Takes a body: https://www.elastic.co/guide/en/elasticsearch/reference/7.x/cluster-nodes-reload-secure-settings.html